dev-lang/python-oem/python-oem-3.9.11.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 2022 Gentoo Authors
```

Comment:
Since this is very different from `python-oem-3.9.8.ebuild` please remove `python-oem-3.9.8.ebuild` to avoid confusion